src/lib/proof_cache_tag/dune

```diff
@@ -1,12 +1,15 @@
 (library
  (public_name proof_cache_tag)
  (virtual_modules proof_cache_tag)
- (default_implementation proof_cache_tag.identity)
+ (default_implementation proof_cache_tag.lmdb)
```

Comment:
Let's not set it as default implementation for now.

Reply:
Let's specify the implementation just for the executables in `src/app/cli/src/dune`